R/xtable.R

```diff
@@ -1,7 +1,9 @@
 # xtable.gsDesign roxy [sinew] ----
+#' @title Tables for ...
```

Comment:
I am not very sure about the functionality of this function and it need reasonable description here.